config/postcss.config.js

```diff
@@ -5,7 +5,8 @@ const postcssCustomProperties = require('postcss-custom-properties');
 const postcssCustomMedia = require('postcss-custom-media');
 
 module.exports = {
-  modules: true,
+  autoModules: false,
```

Comment:
is needed to make `modules` options object work

Reply:
@kuguarpwnz any serious impacts you can imagine?